rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,7 +54,127 @@ paths:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    format:
+                      $ref: '#/components/schemas/MetadataFormat'
+                    location:
+                      description: The URI for where the metadata is located.
+                      type: string
+                      format: uri
+                      minLength: 1
+                      maxLength: 2048
+                      example: 'https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          description: The transformation failed.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+    delete:
+      summary: Delete the transformation request
+      description: >
+        After retrieving the output of the transformation, the client should call this endpoint to remove the request.
+      operationId: delete
+      tags:
+        - transform
+      responses:
+        '204':
+          description: The transformation request was successfully deleted.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+  /transform/{TransformId}/{MetadataFormat}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+      - name: MetadataFormat
+        in: path
+        description: The metadata format ID.
+        required: true
+        schema:
+          $ref: '#/components/schemas/MetadataFormat'
+    get:
+      summary: Retrieve metadata.
+      description: Used to retrieve the metadata for a transform request.
+      operationId: retrieve
+      tags:
+        - transform
+      responses:
+        '200':
+          description: Successfully retrieved.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            text/plain:
```

Comment:
❗️ The response should be the blob for the file we generate for that particular metadata type. Its content type would be whatever is appropriate for that particular file.

Reply:
I went with this solution, based on https://swagger.io/docs/specification/describing-request-body/file-upload/
```
content:
  application/octet-stream:
    schema:
      type: string
      format: binary
      description: The metadata contents.
```
Let me know if you have more specific feedback on how to handle this generically for any given metadata type.